src/components/job/JobApproveReject.jsx

```diff
@@ -40,7 +40,7 @@ export default function JobApproveReject({ jobId }) {
     <>
       <div className="items-center text-center w-3/4">
         <div className="w-full flex flex-row justify-center">
-          <label htmlFor="feedback-display" className="btn btn-primary">
+          <label htmlFor="feedback-display" className="btn btn-secondary">
```

Comment:
sry for not making it clear. here should be a normal button, just `btn` is good

Reply:
no worries, thanks for clarifying. done